src/components/box/pages/box-interactive.jsx

```diff
@@ -47,17 +50,20 @@ const Boxes = () => {
   return (
     <div>
       <DocsActiveBlock settings={settings}>
-        <Box>
+        <Box {...urlParams}>
```

Comment:
???

Reply:
from now on we can pass url params like that: `https://styleguide.brainly.com/latest/docs/interactive.html?shadow=true#boxes` so when you click url in jsdoc and our sg documentation will be opened - it will be already prefilled with prop you click on

Reply:
@PinkaminaDianePie maybe you could add GIF into your PR description to illustrate how it's gonna work more or less?

Reply:
not really, to show proper work we need it to be deployed - when i showed it to you on my machine i changed urls manually and it took a lot of time. i can show it to any interested person on my machine as well :)